.editorconfig

```diff
@@ -0,0 +1,7 @@
+root=true
+
+[*]
+charset=utf8
+indent_style=space
+indent_size=4
+
```

Comment:
oh okay. i never saw this kind of file, and i thought you committed it by accident.